sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/TableResourcesOperationsTests.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    [TestFixture]
+    public class TableResourcesOperationsTests : CosmosDBManagementClientBase
+    {
+        private string resourceGroupName;
+        private const string databaseAccountName = "db2672";
+        private const string tableName = "tableName2527";
+        private const int sampleThroughput = 700;
+        private const int defaultThroughput = 400;
+        private const int defaultMaxThroughput = 4000;
+        public TableResourcesOperationsTests()
+            : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task ClearAndInitialize()
+        {
+            if (Mode == RecordedTestMode.Record || Mode == RecordedTestMode.Playback)
+            {
+                InitializeClients();
+                this.resourceGroupName = Recording.GenerateAssetName(CosmosDBTestUtilities.ResourceGroupPrefix);
+                await CosmosDBTestUtilities.TryRegisterResourceGroupAsync(ResourceGroupsOperations,
+                    CosmosDBTestUtilities.Location,
+                    this.resourceGroupName);
+            }
+        }
+
+        [TearDown]
+        public async Task CleanupResourceGroup()
+        {
+            await CleanupResourceGroupsAsync();
+        }
+
+        [TestCase]
+        public async Task TableCRUDTest()
+        {
+            CosmosDBManagementClient cosmosDBMgmtClient = GetCosmosDBManagementClient();
```

Comment:
Here we don't have to re-initialize the CosmosDBClient. Since we initialize it in the constructor it can just be referred to as CosmosDBManagementClient which we named in the ClientBase.